twarep/electrifly
# UI card

## Checklist

## What I Did
- I changed the layout of all the graphs to make sure that the legends are visible. There will be slight cut-offs here and there. I have spent over 7 hours figuring out why, doesn't make sense. but it shows.
- Restructured the way that the graphs are shown

## How I Did it?
- I installed `faicon` and got the image from there.
- Pure old coding.

## How to Test?
- Just look at the graphs and make sure that you can see the legends, axis, and that there is no error showing.

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
